modules/webbrowser/src/screenspacebrowser.cpp

```diff
@@ -181,7 +178,7 @@ void ScreenSpaceBrowser::render(const RenderData& renderData) {
 }
 
 void ScreenSpaceBrowser::update() {
-    _objectSize = _texture->dimensions();
+    _objectSize = _dimensions.value();
```

Comment:
```suggestion
    _objectSize = _dimensions;
```

Should work as well?

Reply:
Doesn't work :(